### PR TITLE
QRep UI Improvements

### PR DESCRIPTION
**Pull request by Amogh-Bharadwaj**
- Searchable dropdowns for table, watermark column, schema selection
- Source peers in create mirror are only postgres peers
- Fixes mirror status screen

Partially implements #646

### CODE REVIEW

**Review by serprex — APPROVED** [review]

File: ui/package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "protobufjs": "^7.2.5",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-select": "^5.8.0",
```
Comment (serprex) [inline ui/package.json:52]: Why is this necessary given existing dependency on @radix-ui/react-select ?

Comment (Amogh-Bharadwaj) [reply ui/package.json:52]: We want the select dropdowns to be searchable - which radix ui's react select does not seem to have

Comment (serprex) [reply ui/package.json:52]: A separate PR should be made making the switch, removing @radix-ui/react-select too
